docs/deployment/add-semgrep-other-ci.md

```diff
@@ -0,0 +1,147 @@
+---
+slug: add-semgrep-to-other-ci-providers
+title: Add Semgrep to other CI providers
+hide_title: true
+description: "Set up your CI pipeline manually with Semgrep Cloud Platform for centralized rule and findings management."
+tags:
+  - Deployment
+---
+
+import NextStepsComments from "/src/components/concept/_next-steps-comments.mdx"
+
+# Add Semgrep to other CI providers
+
+:::note Your deployment journey
+- You have gained the necessary [resource access and permissions](/deployment/checklist) required for deployment.
+- You have [created a Semgrep account and organization](/deployment/create-account-and-orgs). 
+- For GitHub and GitLab users: You have [connected your source code manager](/deployment/connect-scm).
+- Optionally, you have [set up SSO](/deployment/sso).
+:::
+
+This guide walks you through creating a Semgrep job for CI providers that are **not** listed in Semgrep Cloud Platform (SCP). Skip this guide if you have already configured a CI job.
+
+This method is known to work with the following CI providers:
+
+* AppVeyor
+* Bamboo
+* Bitrise
+* Buildbot
+* Codeship
+* Codefresh
+* Drone CI
+* Nomad
+* TeamCity CI
+* Travis CI
+
+## General steps
+
+The general steps are:
+
+1. Create a `SEMGREP_APP_TOKEN`.
+1. Add this token as a credential, secret, or token into your CI provider and CI configuration file.
+1. For GitHub repositories: Grant permissions for [Semgrep Cloud Platform](https://github.com/marketplace/semgrep-dev).
```

Comment:
Yeah, I think these instructions are outdated. Thanks for the catch. The entire section is actually unnecessary (and I've removed it) because we ask for GH permissions much earlier in the flow.